lib/user.rb

```diff
@@ -0,0 +1,70 @@
+require_relative "workspace"
+require "httparty"
+require "dotenv"
+Dotenv.load
+
+module SlackCLI
+  USER_URL_MESSAGE = "https://slack.com/api/".freeze
+  API_KEY = ENV["SLACK_API_TOKEN"]
+
+  class User
+    attr_reader :name, :real_name, :id
+
+    def initialize(name, real_name, id)
+      @name = name
+      @id = id
+      @real_name = real_name
+    end
+
+    def self.get(url, param)
+      query = { token: param }
+      response = HTTParty.get(url, query: query)
+      response
+    end
+
+    def self.list(url, param)
+      response = get(url, param)
+      members = response["members"]
+      formatted_list = []
+      members.each do |member|
+        name = member["name"]
+        real_name = member["real_name"]
+        id = member["id"]
+        formatted_list << new(name, real_name, id)
+      end
+      formatted_list
+    end
+
+    def self.details(user, url, param)
```

Comment:
This method doesn't make sense as a class method since you want the details of a specific instance of `User`.